eBoard.md

```diff
@@ -13,24 +15,40 @@ The SSE Eboard is committed to bringing you the best SSE experience possible. Fr
 
 ### President - Sabrina Stangler
 I'm Sabrina Stangler, I am the President of SSE for the second year in a row, and I'll be a Senior SE for the 19-20 school year. I work at MacGregor Partners as a Product Development Intern, and I have two lovely cats. I really enjoy the requirements gathering process of software projects and designing GUI layouts. I also quite enjoy interacting with databases on the backend, which is not something I've gotten into a lot, but am eager to learn more about.
+{% assign person = site.eboard.sabrina %}
+{% include contact.html %}
 
 ### View President - Nicholas Scharrer
 Hi there, I am Nick Scharrer and an currently the Vice President of SSE. I formerly was the Administrative Coordinator of SSE, and I'm a Senior as of the 2019-20 school year. I work for Intel currently as a co-op in California, but will be back at MSOE full-time in the Fall of 2019. I formerly worked at two internship sessions with Collins Aerospace (formerly Rockwell Collins), so have some experience working remotely during the school year. I love learning and working on both the hardware and software side of computing, and really enjoy leading teams!
+{% assign person = site.eboard.nick %}
+{% include contact.html %}
 
 ### Communications Coordinator - Brady Cassada
-Hi! I'm Brady Cassada, the current Communications Coordinator of SSE. I am a Junior Software Engineering student at MSOE and I have been interning at Collins Aerospace since last summer. I have a passion for all different types of software and I love expanding my knowledge. In my free time I like playing hockey, making music, and playing video games. 
+Hi! I'm Brady Cassada, the current Communications Coordinator of SSE. I am a Junior Software Engineering student at MSOE and I have been interning at Collins Aerospace since last summer. I have a passion for all different types of software and I love expanding my knowledge. In my free time I like playing hockey, making music, and playing video games.
+{% assign person = nil %}
```

Comment:
Why assign person = nil?
Consider deleting {% assign %} and {% include %} if you don't want to include contact info for them.  This is why contributes to the already crazy high build times this website has.  You can add it back in the future

Reply:
The assignment of person to nil is because I have no information for that person in the config file.  I wanted the page to still include some of the formatting that would come with a entry, and make it easier to add one in the future.

Also, do official builds do a significant amount more than what I do on my computer?  Because I get about a 3 second build there.

Reply:
First of all, I thought this was Grace's pull request, so I was a lot being very blunt/straightforward.  I didn't realize it was someone else's, otherwise I would've elaborated more.  I figured if Grace had a problem with what I said she would just bother me.

One thing I like about removing person=nil and removing the {% if person %} check in the template html is that it fails fast.  So if someone in the future uses it, and it doesn't work, it's VERY obvious that it failed, whether that be incorrect information or a build error.

It's a little weird because why would include a template for a contact if we didn't want to make a contact.  So I saw multiple solutions to this, and I think you should pick whatever you feel is best.  At the end of the day, I won't lose sleep over what you do, and I hope you don't either. You can either:
   1) Comment it out. It's there when we need it in the future. You can then remove the nil check in the template
   2) I don't know if you knew this, but you can pass variables into templates `{% include contact.html person=site.eboard.nick %}`, and then have a separate variable you can optionally set called "empty" or something. Just so we have clearly defined behavior, and the template fails if we use it incorrectly
  3) Lastly, you keep it the same. If you really feel passionate about doing it this way, then I won't stop you

Reply:
I was unaware that variables could be passed directly, I'll probably use that approach.